### ISSUE TITLE
Weird issues with batches larger than 1

### ISSUE THREAD

**Issue by bernardopires** [kuprel/skin#1]
Hi Brett,

I came across your repo when I saw your post at tensorflow google group about retraining inception and was really excited that I'd be able to finetune the whole network.

I have this weird issue with batches larger than 1 which I can not pinpoint exactly where it is happening. I basically reused your code for loading inception v3 and just use a different loss function. I trained a model and it seems like everything went fine. But when evaluating the images, i.e. not training, so simply doing a forward pass, every time I evaluate I get different results, by a relatively large amount (up to 20%) even though it's always exactly the same images. When using `batch_size=1` the results are consistently the same (minus tiny differences due to the batchnorm layer). What I find really weird is that if I use for example a batch of 16 images, and all 16 images are the same, then I consistently get the same results out, just as if I had a batch_size of 1. So it seems like somehow the images inside a batch are affecting the others.

Did you notice anything like this? Can you please check if you can reproduce this? Thanks a lot!

Cheers,
Bernardo
